src/main/com/yetanalytics/datasim/clock.clj

```diff
@@ -1,5 +1,5 @@
 (ns com.yetanalytics.datasim.clock
-  (:require [java-time :as t])
+  (:require [java-time.api :as t])
```

Comment:
This is done because the `java-time` namespace is deprecated in favor of `java-time.api` in the newer versions of the library.